libc/include/llvm-libc-types/float128.h

```diff
@@ -9,7 +9,7 @@
 #ifndef __LLVM_LIBC_TYPES_FLOAT128_H__
 #define __LLVM_LIBC_TYPES_FLOAT128_H__
 
-#include <include/llvm-libc-macros/float-macros.h> // LDBL_MANT_DIG
+#include <llvm-libc-macros/float-macros.h> // LDBL_MANT_DIG
```

Comment:
This should never have been `<>`. While you're here, please change this to `""`.

Reply:
The other headers seem to use `<>` in stuff like `math.h.def`.

Reply:
Check my grep command.  `libc/include/llvm-libc-types/float128.h` is the ONLY place where `<>` is used to include float-macros.h. Every other include uses `""`.  Consistency is good.

Reply:
all the `#include`s in `/include/` for things in `llvm-libc-types` or `llvm-libc-macros` currently use `<>`. It may be worthwhile to change this, but that should probably be one cleanup patch.

Reply:
Filed: https://github.com/llvm/llvm-project/issues/83210